docs/pages/includes/server-access/custom-installer.mdx

```diff
@@ -0,0 +1,84 @@
+{{ cloud="foo" matcher="bar" matchTypes="baz" }}
+To customize the default installer script, execute the following command on
+your workstation:
+
+```code
+$ tctl get installer/default-installer > teleport-default-installer.yaml
+```
+
+The resulting `teleport-default-installer.yaml` can be edited to
+change what gets executed when enrolling discovered instances.
+
+After making the desired changes to the default installer, the
+resource can be updated by executing:
+
+```code
+$ tctl create -f teleport-default-installer.yaml
+```
+
+Multiple `installer` resources can exist and be specified in the
+`{{ matcher }}.install.script_name` section of a `discovery_service.{{ matcher }}` list item in
+`teleport.yaml`:
+
+```yaml
+discovery_service:
+  {{ matcher }}:
+    - types: {{ matchTypes }}
+      tags:
+       - "env": "prod"
+      install: # optional section when default-installer is used.
+        script_name: "default-installer"
+    - types: {{ matchTypes }}
+      tags:
+       - "env": "devel"
+      install:
+        script_name: "devel-installer"
+```
+
+---
+
+The `installer` resource has the following templating options:
+
+- `{{ .MajorVersion }}`: the major version of Teleport to use when
+  installing from the repository.
+- `{{ .PublicProxyAddr }}`: the public address of the Teleport Proxy Service to
+connect to.
+- `{{ .RepoChannel }}`: Optional package repository (apt/yum) channel name.
+Has format `<channel>/<version>` e.g. stable/v(=teleport.major_version=). See [installation](../../installation.mdx#linux) for more details.
+- `{{ .AutomaticUpgrades }}`: indicates whether Automatic Upgrades are enabled or disabled.
+  Its value is either `true` or `false`. See
+  [self-hosted automatic agent updates](../../management/operations/self-hosted-automatic-agent-updates.mdx)
+  for more information.
+- `{{ .TeleportPackage }}`: the Teleport package to use.
+  Its value is either `teleport-ent` or `teleport` depending on whether the
+  cluster is enterprise or not.
+
+These can be used as follows:
+
+```yaml
+kind: installer
+metadata:
+  name: default-installer
+spec:
+  script: |
+    echo {{ .PublicProxyAddr }}
+    echo Teleport-{{ .MajorVersion }}
+    echo Repository Channel: {{ .RepoChannel }}
+version: v1
+```
+
+Which, when retrieved by the Systems Manager document will evaluate to a script
+with the following contents:
```

Comment:
This is an AWS Service name.
Does it have the same name in Azure?
If not, then I would say we should come up with a generic name for both (to avoid duplicating the guide's text).

Not blocking because this was already like that 👍

Reply:
good point - no it does not have the same name in Azure

I reworded it to be generic